docker/Dockerfile

```diff
@@ -36,9 +36,8 @@ RUN apt update -y && apt install -y \
 
 RUN apt install -y python3-pprintpp; exit 0
 RUN apt clean -y all
-
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 10
```

Comment:
We've always relied on users doing this to ensure that a "bare" `python` is on the path.  Looking at all your updates for MacOS, I wonder if it's best practice to always specify `python3`.  It seems clunky to me, but maybe too many systems keep python 2.x around as `python`?

Reply:
I agree, I think it would be better to not rely on a "bare" `python`.  I initially removed that line in the Dockerfile, but it will require appropriate adjustments in the downstream projects.  Given that there are other changes required for the downstream projects I will remove it again, and make PRs in cycamore and cymetric and we can merge them in sync.